tests/test_verify_base_url.py

```diff
@@ -20,6 +22,7 @@ def test_ignore_bad_url_by_default(testdir, httpserver):
 def test_enable_verify_via_cli(testdir, httpserver, monkeypatch):
     testdir.makepyfile('def test_pass(): pass')
     monkeypatch.setenv('VERIFY_BASE_URL', False)
+    monkeypatch.setattr(Retry, 'BACKOFF_MAX', 5)
```

Comment:
In my testing, we can set this much lower. I used `0.5` and it took around 4 seconds to raise for each test.